staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -74,6 +74,9 @@ const (
 type KubeletConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// enableServer enables the Kubelet's server
```

Comment:
clarify which server this enables (the secured port, I think). might also be worth mentioning that the readonly port is independently controlled.